api/Dockerfile

```diff
@@ -3,7 +3,7 @@ WORKDIR /tmp
 
 # Note this step requires docker builds in the root repo context
 # e.g. `docker build -t wfl-api:test -f ./api/Dockerfile .`
-ADD .. .
+ADD . .
```

Comment:
<img width="1399" alt="Screen Shot 2021-04-12 at 2 44 04 PM" src="https://user-images.githubusercontent.com/14366016/114445575-3b7b3380-9b9e-11eb-8d0a-4a79f473d223.png">

I'm still puzzled by the issue the Action is reporting, locally, from with the root `make api TARGET=images` works both before and after the change. Let's see if this fixes the Action :octocat:

Reply:
Interesting. This failed on my machine. Did the docker build actually run when you tried the before and after?

Reply:
I believe so.. I ran both `make distclean` and `docker system prune -a` before `make api TARGET=images` 🤔 I can double check.

Reply:
How did that ever work then?

Reply:
Honestly I have no idea. Different versions of docker perhaps? I never thought you could add the parent directory.